pallets/pdex-migration/src/benchmarking.rs

```diff
@@ -1,45 +1,38 @@
-//! Benchmarking setup for pallet-template
-
-use frame_benchmarking::{account, allowlisted_caller, benchmarks, impl_benchmark_test_suite};
-use frame_support::{assert_ok, pallet_prelude::*, traits::Get};
-use frame_system::{pallet_prelude::*, RawOrigin};
-use rand::{RngCore, SeedableRng};
-// use frame_system::Origin;
-use sp_core::H256;
-use sp_runtime::{traits::BlockNumberProvider, SaturatedConversion};
-
 use crate::pallet::{Call, Config, Pallet as PDEXMigration, Pallet, *};
+use frame_benchmarking::{account, benchmarks, whitelisted_caller};
+use frame_support::{assert_ok, dispatch::UnfilteredDispatchable, traits::Get};
+use frame_system::RawOrigin;
+use sp_runtime::{traits::BlockNumberProvider, SaturatedConversion};
 
-use super::*;
+const PDEX: u128 = 1000_000_000_000;
 
-// use crate::pallet::Call as PDEXMigration;
-// use crate::mock::{
-//     // PDEXMigration,
-//     PDEX
-// };
-pub const PDEX: u128 = 1000_000_000_000;
 benchmarks! {
 	set_migration_operational_status {
-
-	}: _(RawOrigin::Root, true)
+		let call = Call::<T>::set_migration_operational_status { status: true };
+	}: { call.dispatch_bypass_filter(RawOrigin::Root.into())? }
+	verify {
+		assert!(<Operational<T>>::get());
+	}
 
 	set_relayer_status {
-		let relayer : T::AccountId = account("relayer",0,0);
-	}: _ (RawOrigin::Root, relayer, true)
+		let relayer : T::AccountId = account("relayer", 0, 0);
+		let call = Call::<T>::set_relayer_status { relayer: relayer.clone(), status: true };
+	}: { call.dispatch_bypass_filter(RawOrigin::Root.into())? }
+	verify {
+		assert!(<Relayers<T>>::get(relayer));
+	}
 
 	mint {
+		let b in 1 .. 256;
 		let relayer1: T::AccountId = account("relayer1",0,0);
 		let relayer2: T::AccountId = account("relayer2",0,0);
 		let relayer3: T::AccountId = account("relayer3",0,0);
-		let beneficiary: T::AccountId  = allowlisted_caller();
+		let beneficiary: T::AccountId  = whitelisted_caller();
```

Comment:
Mixed terminology, in all places used `allowlist` but here I assume outdated `whitelist`.

Reply:
this function is part of substrate's [frame-benchmarking](https://github.com/paritytech/substrate/blob/ac176294e647f0eabd0970537567bf37ded00360/frame/benchmarking/src/utils.rs#L365) crate, which is not developed by us.
if anything - all other methods should be renamed to use `whitelist` prefix. but under separate issue :)